submissions/adeyemiJuwon/javascript/loop.js

```diff
@@ -0,0 +1,20 @@
+let a = 123;
+  let b = 7;
+  let c = a - b;
+
+for (var i = 1; i < c; i++){
+
+    if ( i % 3 == 0) {
+        console.log('Software');
+    
+    }else if (i % 5 == 0){
+        console.log('Developer');
+    
+    }else if (i % 3 == 0 && i % 5 == 0){
```

Comment:
So, with the way you did this, you won't have a case where it will log out `software developer`. Because the other two conditions must have been met first before it gets here. Say, for example, number `15`. This number ought to log out `software developer`. But because this `i % 3 == 0` is the first inline that will be executed first. It won't bother executing other conditions. That is what `if-else and elseif` statement does. 

Having this in mind, can you now refactor your code to give a better result?

Reply:
Yes
Thank you

Reply:
You still haven't fix it @Samju200

Reply:
@adaugochi i loop the for (i % 3 == 0 && i % 5 == 0) first which can now print three condition given
![ada](https://user-images.githubusercontent.com/66218712/92099384-56005980-edd2-11ea-92a5-aa4ed9d51f6e.png)

 (var i = 1; i < c; i++){

    if (i % 3 == 0 && i % 5 == 0){
        console.log('Software Developr');
    
    }else if ( i % 3 == 0) {
        console.log('Software');
    
    }else if (i % 5 == 0){
        console.log('Developer');
    
    }else{
        console.log(i);
    
    }
    }

Reply:
@adaugochi i just recheck it now, my change did not reflect before, that is why.